commit 25fcfc583d3fe1be6d9a3455933d12b2a5bd5fbd
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2019-04-09T12:13:41-07:00 (6 months ago)
    
    TRITON-1379 When server_reboot fails, CNAPI does not deal with it well
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/lib/endpoints/servers.js b/lib/endpoints/servers.js
index 4e0a500..9ee47dc 100644
--- a/lib/endpoints/servers.js
+++ b/lib/endpoints/servers.js
@@ -651,7 +651,9 @@ Server.reboot = function handlerServerReboot(req, res, next) {
                 });
            }
         ]}, function _doneReboot(err) {
-            res.send(204);
+            if (!err) {
+                res.send(204);
+            }
             next(err);
         });
         return;
diff --git a/package.json b/package.json
index d8b93e0..bb26798 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "Triton Compute Node API",
-  "version": "1.23.2",
+  "version": "1.23.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
